test/WebJobs.Script.Tests/TestScripts/Node/HttpTriggerExpressApi/index.js

```diff
@@ -0,0 +1,20 @@
+﻿var util = require('util');
+
+module.exports = function (context, req) {
+    context.log('Node.js HttpTrigger function invoked.');
+
+    context.res.status(200)
```

Comment:
yeah, I'll expand this, accepts and get are definitely useful.  Correct me if I'm wrong, but we don't support params for httptriggers (params like `functionurl/:id`, not query params)